impl/pom.xml

```diff
@@ -440,18 +427,4 @@ Copyright &#169; 2019 Eclipse Foundation. All rights reserved.
             </plugin>
         </plugins>
     </build>
-
-    <reporting>
-        <plugins>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <version>${findbugs.version}</version>
-                <configuration>
-                    <threshold>${findbugs.threshold}</threshold>
-                    <excludeFilterFile>${findbugs.exclude}</excludeFilterFile>
-                </configuration>
-            </plugin>
-        </plugins>
-    </reporting>
```

Comment:
Why are we removing the findbugs plugin here?

Reply:
Less noise, smaller poms. IMHO finding bugs can much better be done by tools like sonar cloud.